scripts/build.sh

```diff
@@ -27,6 +27,7 @@ LLVM_VERSION=llvm-9
 OS_VERSION=
 ARCH_VERSION=
 BUILD_FLAGS=
+CXX_COMMON_VERSION="v0.1.1"
```

Comment:
I really should have done this at the beginning. Hopefully it wasn't too much headache 😬 

I _think_ `v0.1.2` should be all done now. Can we try that bump here?